xml/cap_depl_install_production.xml

```diff
@@ -434,6 +435,28 @@ clusterrolebinding.rbac.authorization.k8s.io "cap:clusterrole" created</screen>
         <literal>suse.cap.psp</literal>.
     </para>
 </tip>
+   <important>
+    <title>Using Custom PodSecurityPolicies</title>
+    <para>
+     When using a custom PSP, your <filename>scf-config-values.yaml</filename> file will require the <literal>SYS_RESOURCE</literal> capability to be added to some roles as follows:
+    </para>
+<screen>sizing:
```

Comment:
`sizing:` can go to the next line and `<screen>` can be indented to match the `</para>`, indicating it is a child of `<important />`.